packages/desktop-client/src/components/LoggedInUser.tsx

```diff
@@ -16,15 +13,16 @@ import { useServerURL } from './ServerContext';
 
 type LoggedInUserProps = {
   hideIfNoServer?: boolean;
+  syncState: null | 'offline' | 'local' | 'disabled' | 'error';
   style?: CSSProperties;
   color?: string;
 };
 export function LoggedInUser({
   hideIfNoServer,
+  syncState,
```

Comment:
Yes, I should be able to do this there as well.

Reply:
Ok so after looking into this area of the code, it definitely won't work.

This is making me rethink how that button works in general, and the approach used.  Perhaps the button needs its own logic to determine the state instead of relying on other components.

Reply:
This is currently working in the latest master so if we merge this branch we'd be breaking it.

Reply:
Right, like I said it probably needs reworked.